ecs-cli/modules/cli/configure/migrate.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package configure
+
+import (
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+)
+
+func maskSecret(secret string) (out string) {
+	if secret != "" {
+		mask := strings.Repeat("*", len(secret)-2)
+		out = strings.Replace(secret, secret[:len(secret)-2], mask, 1)
+	}
+	return out
```

Comment:
oops, just realized that strictly speaking the "out" isn't needed here, but that's just a caviling detail.

Reply:
Yeah, I know, I actually kind of like named arguments though ;) so I kept it in when I copied your code snippet.